asterisk_click2dial/models/asterisk_server.py

```diff
@@ -25,7 +25,7 @@ class AsteriskServer(models.Model):
     ip_address = fields.Char(string="Asterisk IP address or DNS", required=True)
     port = fields.Integer(
         required=True,
-        default=5038,
+        default=8088,
```

Comment:
default port for AMI is 5038 indeed
8088 is for the web interface

Reply:
AMI is no longer used. It was changed on v14

Reply:
OMG, time goes fast